test/drbgtest.c

```diff
@@ -588,7 +588,8 @@ static int test_drbg_reseed(int expect_success,
                             RAND_DRBG *private,
                             int expect_master_reseed,
                             int expect_public_reseed,
-                            int expect_private_reseed
+                            int expect_private_reseed,
+                            time_t reseed_time
```

Comment:
The `reseed_time` needs to be documented in the doc comment. How about something like the following?

```
* |reseed_time|: if nonzero, used instead of time(0) to set the |before_reseed| time.
```

Reply:
OK.  Will just change time(0) to time(NULL).

Reply:
Right. Bad habit of a C++ programmer...

Reply:
...  I've never seen a C compiler that won't accept 0 instead of `NULL`.  Bjarne got rid of `NULL` because it really wasn't necessary.  But hey, conventions...